dev/tidb-in-kubernetes/deploy/aws-eks.md

```diff
@@ -331,6 +331,9 @@ module example-cluster {
   monitor_instance_type         = "t2.xlarge"
   # The version of tidb-cluster helm chart
   tidb_cluster_chart_version    = "v1.0.0"
+  # Wether creating the tidb-cluster helm release, if this variable set to false, you have to
+  # install the helm release manually
+  create_tidb_cluster_release   = true
```

Comment:
Where is this variable used to control the installation of tidb cluster?

Reply:
https://github.com/pingcap/tidb-operator/pull/1062 fix this